source/simulator/solver.cc

```diff
@@ -267,7 +289,7 @@ namespace aspect
       // first solve with the bottom left block, which we have built
       // as a mass matrix with the inverse of the viscosity
       {
-        SolverControl solver_control(1000, 1e-6 * src.block(1).l2_norm());
+        SolverControl solver_control(1000, src.block(1).l2_norm() * S_block_tolerance);
```

Comment:
Please unify the use of spaces here and in the next change. I know it was different before, but this is our change to change it ;-). I like spaces in such cases, but this is just my personal opinion.